src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Bulldozer(),new Excavator(),new Truck()};
```

Comment:
and some code style tweaks :)
```suggestion
        Machine[] machines = new Machine[] {new Bulldozer(), new Excavator(), new Truck()};
```